test/notebook-tests.py

```diff
@@ -201,93 +200,123 @@ def test_t5():
+    source = os.path.join("../test/test_output", mlp.project.ml_folder, "images")
+    save_dir = project_path
+    conf_thres = t_utils.choose_conf()
+    mlp.detect_yolov5(
+        exp_name=exp_name,
+        source=source,
+        save_dir=save_dir,
+        conf_thres=conf_thres.value,
+        artifact_dir=artifact_dir,
+    )
+    eval_dir = exp_path
+    mlp.save_detections_wandb(conf_thres.value, model, eval_dir)
+
+    # Create unique tracking experiment name
+    track_exp_name = (
+        f"tracker_test_{dt}".replace(" ", "_").replace(".", "_").replace(":", "-")
+    )
+
+    # Tracking individuals
+    mlp.track_individuals(
+        name=track_exp_name,
+        source=source,
+        artifact_dir=artifact_dir,
+        eval_dir=eval_dir,
+        conf_thres=conf_thres.value,
+        img_size=(640, 640),
+    )
+    shutil.rmtree(exp_path)
 
 
 # #-------------Tutorial 7-------------------------------------------------------
-# """
-# ...
-# """
+"""
+This tutorial is still in a very early stage and mostly uses widgets directly from widgets.py and
+API functionality from Zenodo. Tests should be added in future as its capabilities expand. 
+"""
 
-# def test_t7():
-# model = mlp.choose_model()
-# download_dir = kso_widgets.choose_folder(".", "downloaded model")
-# artifact_dir = mlp.get_model(model.value, download_dir.selected)
-# ACCESS_TOKEN = ""
-# archive_dir = kso_widgets.choose_folder(".", "archive for upload")
-# depo_id = zenodo_utils.upload_archive(ACCESS_TOKEN, artifact_dir=artifact_dir)
-# upload_title = t_utils.choose_text("title")
-# upload_description = t_utils.choose_text("description")
-# authors = t_utils.WidgetMaker()
-# zenodo_utils.add_metadata_zenodo_upload(
-#     ACCESS_TOKEN,
-#     depo_id,
-#     upload_title.value,
-#     upload_description.value,
-#     authors.author_dict,
-# )
-
-
-# #-------------Tutorial 8-------------------------------------------------------
-# """
-# ...
-# """
-# pp.choose_workflows(False, zoo_cred=zoo_cred)
-
-# workflow_checks = {'Workflow name: #0': 'Development workflow',
-#  'Subject type: #0': 'clip',
-#  'Minimum workflow version: #0': 1.0}
-
-# class_df = pp.get_classifications(
-#     workflow_checks,
-#     pp.zoo_info["workflows"],
-#     workflow_checks["Subject type: #0"],
-#     pp.zoo_info["classifications"],
-# )
+
+def test_t7():
+    pass
+
+
+# -------------Tutorial 8-------------------------------------------------------
+"""
+...
+"""
+
+
+def test_t8(zoo_user, zoo_pass):
+    import kso_utils.widgets as kso_widgets
+
+    # Connect to zooniverse w/ Github credential
+    pp.get_zoo_info(zoo_cred=[zoo_user, zoo_pass])
+    pp.choose_workflows(generate_export=False, zoo_cred=[zoo_user, zoo_pass])
+
+    workflow_checks = {
+        "Workflow name: #0": "Development workflow",
+        "Subject type: #0": "clip",
+        "Minimum workflow version: #0": 1.0,
+    }
+
+    class_df = pp.get_classifications(
```

Comment:
I do not see this one in the dev version of NB 8. Maybe add a comment where it is?

Because in all the other tests, it is quite easy to follow the test function with the notebook next to it. So that is great! But here someone could get a bit confused since it is not in the NB (or I just don't see it).